### PR TITLE
Community steps

### PR DESCRIPTION
**Pull request by dcbroad3**

### CODE REVIEW

File: Sample.js [hunk 1]
```diff
@@ -1984,10 +1985,146 @@ var app = function (request1, response) {
...
+            .catch(function (err) {
+              logError(err);
+            });
+        } else {
+          return client.getTenantRoles(tenantId);
+        }
+      }
+    );
+
+    var communityMemberRoleTypeId = 'f79a55da-7c76-4600-a809-0f62ca9971d9';
```
Comment (derekendres) [inline Sample.js:2011]: constant vars like this in most languages go at the top and designated with different naming is that true here?

**Review by derekendres — APPROVED** [review]

File: Sample.js (hunk 1 above)
Comment (kelseyduffy) [reply Sample.js:2011]: such as my favorite one, SCREAMING_SNAKE_CASE :D

**Review by kelseyduffy — APPROVED** [review]

File: Sample.js (hunk 1 above)
Comment (dcbroad3) [reply Sample.js]: Good point, moved this to the top

**Pull request merged**